android/src/main/java/com/instabug/flutter/modules/ApmApi.java

```diff
@@ -213,4 +221,71 @@ public void networkLogAndroid(@NonNull Map<String, Object> data) {
             e.printStackTrace();
         }
     }
+
+
+    @Override
+    public void startCpUiTrace(@NonNull String screenName, @NonNull Long microTimeStamp, @NonNull Long traceId) {
+        try {
+            InternalAPM._startUiTraceCP(screenName, microTimeStamp, traceId);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Override
+    public void reportScreenLoadingCP(@NonNull Long startTimeStampMicro, @NonNull Long durationMicro, @NonNull Long uiTraceId) {
+        try {
+            InternalAPM._reportScreenLoadingCP(startTimeStampMicro, durationMicro, uiTraceId);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Override
+    public void endScreenLoadingCP(@NonNull Long timeStampMicro, @NonNull Long uiTraceId) {
+        try {
+            InternalAPM._endScreenLoadingCP(timeStampMicro, uiTraceId);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Override
+    public void isEndScreenLoadingEnabled(@NonNull ApmPigeon.Result<Boolean> result) {
+        isScreenLoadingEnabled(result);
+    }
+
+    @Override
+    public void isEnabled(@NonNull ApmPigeon.Result<Boolean> result) {
+        try {
+            // TODO: replace true with an actual implementation of APM.isEnabled once implemented
```

Comment:
there is a TODO statement

Reply:
This TODO statement was set previously and is not related to the screen loading implementation.